app/components/sign-in-button.tsx

```diff
@@ -20,7 +20,7 @@ export default function SignInButton({ large }: { large?: boolean }) {
 
   return (
     <Button asChild size={large ? '3' : '2'}>
-      <a href={authorizationUrl}>Sign In {large && 'with AuthKit'}</a>
+      <a href={signInUrl}>Sign In {large && 'with AuthKit'}</a>
```

Comment:
```suggestion
      <a href={signInUrl}>Sign In{large && ' with AuthKit'}</a>
```
Not related to this PR but a small detail I noticed!